GNUmakefile.osx

```diff
@@ -102,6 +102,7 @@ endif
 
 ifndef LIBXML2_LIBS
     LIBXML2_LIBS := $(shell xml2-config --libs)
+    LIBXML2_LIBS := $(filter-out -L$(shell xcrun --show-sdk-path)/usr/lib, $(LIBXML2_LIBS))
```

Comment:
Oh I just noticed that this is in the osx version of the Makefile, so I guess it's an osx specific command.  Still curious about what this change is supposed to do.